graphql/graphql_test.go

```diff
@@ -123,13 +133,15 @@ func TestQueries(t *testing.T) {
 			AllBugs struct {
 				PageInfo models.PageInfo
 				Nodes    []struct {
-					Author    Person
-					CreatedAt string `json:"createdAt"`
-					HumanId   string `json:"humanId"`
-					Id        string
-					LastEdit  string `json:"lastEdit"`
-					Status    string
-					Title     string
+					Author       Person
```

Comment:
that's a place I forgot to update when I renamed Person to Identity. Would you mind updating it here as well ?

Reply:
I'll make that in my next changes

Reply:
using `identity.Interface` instead of `Person` causes graphql tests to fail. I get some like 
```bash
'DefaultRepository.AllBugs.Nodes[1].Author' expected type 'identity.Interface', got 'map[string]interface {}'
```
maybe we should use a special model struct for identity ?

Reply:
Hoo, I got what you are doing. `Person` here is just a dumb struct (a few lines up) to store the graphql response. I just wanted to rename this struct to `Identity`, not using `identity.Interface`